src/main/java/seedu/duke/controllers/StaffController.java

```diff
@@ -40,21 +47,53 @@ protected boolean optionSwitcher(int choice) throws OperationTerminationExceptio
         return false;
     }
 
-    public Staff findStaff() throws OperationTerminationException {
+    /**
+     * Find staff by ID.
+     *
+     * @throws OperationTerminationException When user inputs terminator.
+     */
+    private void findStaff() throws OperationTerminationException {
+        logInfo("Finding staff");
         System.out.println("Finding staff...");
         int staffId = InputParser.getInteger("ID of staff: ");
-        return staffManager.findByStaffId(staffId, true);
+        assert staffId > 0 : "Invalid ID";
+        staffManager.findByStaffId(staffId, true);
     }
 
-    public void addStaff() throws OperationTerminationException {
+    /**
+     * Add staff.
+     *
+     * @throws OperationTerminationException When user inputs terminator.
+     */
+    private void addStaff() throws OperationTerminationException {
+        logInfo("Adding staff");
         System.out.println("Adding new staff...");
         final int staffId = InputParser.getInteger("ID of staff: ");
+        assert staffId > 0 : "Invalid ID";
```

Comment:
User might crash the whole program here if we use assert?

Reply:
^

assert is just to confirm that the program works, not for error checking, so need to also add a check beforehand to make sure that the staffId will never be 0 or less, THEN u add the assert statement after that.

Reply:
Yeah if we enable assertion it will crash... that's why I'm not exactly sure what's the point of having assertions in our program lol

Reply:
you are confusing assertions with error checking. assertions is to make sure that your program works because you coded it in a way that way such that a wrong state will never be reached. 

in this case you are making sure that staffId will never be <= 0, so you actually need to add a check before the assert statement to make sure that stadffId is never <= 0, and then do whatever is required (such as showing telling user wrong id input) if it ever is.

Reply:
assert statement is for the coder, exception handling is for the user.

assert statement make sure your exception handling works (or just that your program will never reach a state you never intended it to be in the first place).

Reply:
this is also why you can disable assertions for releases, because its meant for the coder, not the user.

Reply:
Oh ok I see I will make the changes